tar5.4_tool.py

```diff
@@ -0,0 +1,42 @@
+"""
+    fanction that return thee min's lengh of the argument
```

Comment:
1. Document inside the function.
2. There are several spelling and grammer mistakes, please fix it.